README.md

```diff
@@ -38,6 +38,16 @@ usethis::use_github_action("lint")
 lintr::lint_package()
```

Comment:
Do we want to retain this in `README`, or can we get rid of it? Not sure if this is still active.

<img width="382" alt="Screenshot 2022-09-30 at 18 49 48" src="https://user-images.githubusercontent.com/11330453/193318991-fab35f0a-3737-46c3-8a40-c90745b1a50f.png">

Reply:
I think we can remove it, AFAIK neither @AshesITR nor I are checking that so not sure how helpful it could be.

Reply:
I have removed it.